controls/raddatafilter/datafilter-getting-started.md

```diff
@@ -70,9 +68,7 @@ You can configure __RadDataFilter__ via the properties that it exposes. You can
 
 ## Filtering a Collection
 
->
-
-To learn more about the usage of the __Source__ and the __FilteredSource__ properties, please read [this topic]({%slug raddatafilter-features-source-and-filtered-source%}).
+> To learn more about the usage of the __Source__ and the __FilteredSource__ properties, please read [this topic]({%slug raddatafilter-features-source-and-filtered-source%}).

```

Comment:
Remove the word 'please' from the sentence. Also set a meaningful name of the link instead of 'this topic'. For example 'read the Source and Filtered Source article.'